packages/gitbook/src/lib/api.ts

```diff
@@ -405,18 +424,35 @@ export const getRevision = cache({
             }
         );
 
-        return cacheResponse(response, fetchOptions.metadata ? cacheTtl_7days : cacheTtl_1day);
+        return cacheResponse(response, {
+            ...(fetchOptions.metadata ? cacheTtl_7days : cacheTtl_1day),
+            data: {
+                ...response.data,
+                tags: getResponseCacheTags(response),
+            },
+        });
+    },
+    getKeyArgs: ([spaceId, revisionId, fetchOptions]) => {
+        if (fetchOptions.computed === false) {
```

Comment:
I think it's opposite; you want to use `[spaceId, revisionId]` when `computed = false`, because that's the first call we do; and we want to re-use existing existing cache.

Otherwise here, the call with `computed: false`, will try a new cache key, and bust the cache for all spaces.